### ISSUE TITLE
GoBridge macos support?

### ISSUE THREAD

**Issue by zbpan-sh** [Xinyuan-LilyGO/T-FPGA#5]
Does the gobridge chip work on macos(M1)？

**Comment by lewisxhe** [comment]
No, the GoWinSemi documentation center shows that there is no software available for Mac.

https://www.gowinsemi.com/en/support/database/14/
